unstructured_inference/inference/layoutelement.py

```diff
@@ -301,8 +306,9 @@ def merge_text_regions(regions: List[TextRegion]) -> TextRegion:
     max_y2 = max([tr.y2 for tr in regions])
 
     merged_text = " ".join([tr.text for tr in regions if tr.text])
-
-    return TextRegion(min_x1, min_y1, max_x2, max_y2, merged_text)
+    sources = [*{tr.source for tr in regions}]
```

Comment:
hmm, is this first get a set then convert to list? maybe `list({tr.source for tr in regions})` would be clearer?